### PR TITLE
[BPK-2481] add grey theming

### PR DESCRIPTION
**Pull request by tiagoengel**
PR for the App's codebase here: https://github.skyscannertools.net/apps-tribe/skyscanner-app/pull/5385

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/THEMING.md [hunk 1]
```diff
@@ -23,29 +23,29 @@ You can theme all instances of `BpkSwitch` in your app/activity by setting the `
 
 __For supported attributes check each component's documentation.__
 
-## Theme overlays and runtime theming
```
Comment (tiagoengel) [inline docs/THEMING.md]: This has been removed. Docs were outdated

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:b1e6f44754ce32840c92192467649fddb56a3b88`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:df71fa1c467d347a2971ae12289cb2952022adf4`

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:62254ca14af297e1e9f9a92dd63b1e3fbd4d8156`

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/ThemesUtilTest.kt [hunk 2]
```diff
@@ -16,34 +18,98 @@ import org.junit.runner.RunWith
...
+  private val gray700 by lazy { ContextCompat.getColor(activity, R.color.bpkGray700) }
+  private val gray900 by lazy { ContextCompat.getColor(activity, R.color.bpkGray900) }
+
+  private val customBlue = Color.parseColor("#9B104A")
+  private val customGrey50 = Color.parseColor("#9B104B")
+  private val customGrey100 = Color.parseColor("#9B104C")
+  private val customGrey300 = Color.parseColor("#9B104D")
+  private val customGrey500 = Color.parseColor("#9B104E")
+  private val customGrey700 = Color.parseColor("#9B104F")
+  private val customGrey900 = Color.parseColor("#9B105A")
```
Comment (shaundon) [inline Backpack/src/androidTest/java/net/skyscanner/backpack/util/ThemesUtilTest.kt]: We should standardise on the spelling here - these lines use 'grey' but the earlier ones use 'gray'. (gray is American spelling, grey is British). We use the US spelling everywhere else so let's go with 'gray'.

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:2cac3271322b92c0202b42087ed0bf2b999790b7`

File: Backpack/src/androidTest/java/net/skyscanner/backpack/util/ThemesUtilTest.kt (hunk 2 above)
Comment (tiagoengel) [reply Backpack/src/androidTest/java/net/skyscanner/backpack/util/ThemesUtilTest.kt]: Done

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:9592bcc887dfd5d8b44658376b1f476d351ba855`

**Pull request merged**